src/data/orders/data.ts

```diff
@@ -221,15 +221,6 @@ export enum OrderStatus {
   pending = 'pending',
 }
 
-export interface OrderAddress extends Address {
```

Comment:
Did we drop accidentally drop
- `organizationName`
- `email`
- `phone`

by this refactor?

Reply:
good catch. change reverted.